src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -318,7 +318,10 @@ public void Dispose()
             }
             if (_intermediateFolder != null)
             {
-                // to-do: add check for build status. if failed, not overwrite buildinfo
+                if (_currentBuildInfo.Versions.Any(v => v.Status == BuildStatus.Failed))
```

Comment:
yes, this could be a problem for the case that build failed. might need to store the output to intermediate files

---
In reply to: [76353748](https://github.com/dotnet/docfx/pull/638#discussion_r76353748) [](ancestors = 76353748)